tests/testthat/test-drive_trash.R

```diff
@@ -0,0 +1,49 @@
+context("Trash files")
+
+## WARNING: this will empty your drive trash. If you do
```

Comment:
Move this down near the actual test and reword to reflect current situation. It's a bit alarming right now, since the test is actually commented out.